src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -92,25 +144,35 @@ public async Task<CommandOrCodeActionContainer> Handle(CodeActionParams request,
                 }
             }
 
+            if (cancellationToken.IsCancellationRequested)
```

Comment:
This needed here? I don't see any async call happening(we don't await anything) since the previous check.